azavea/azavea-branding-guide
# Add clarity to downloads and product/open source pages

## Overview
This PR attempts to address these main issues:
1) People most frequently use the Branding Guidelines for logo downloads, but the logo downloads were spread out amongst 3 pages.
2) The difference between "Products" and "Open Source" is murky, and some branded projects were incorrectly placed, regardless.
3) There is a page called "Logo" that only contains Azavea logos

To solve the first issue, I added a top-level "Logo Downloads" page that simply contains downloads for all the logos.

### Demo
Logo Downloads page
<img width="1792" alt="screen shot 2019-02-27 at 5 30 45 pm" src="https://user-images.githubusercontent.com/5672295/53527609-705ac800-3ab5-11e9-8910-8e2983e372cd.png">

Logo Guidelines page
<img width="1792" alt="screen shot 2019-02-27 at 5 14 17 pm" src="https://user-images.githubusercontent.com/5672295/53527029-c6c70700-3ab3-11e9-9b78-4108ddd4635d.png">

New navigation under "Company"... note that "Open Source" is no longer there.
<img width="1792" alt="screen shot 2019-02-27 at 5 14 23 pm" src="https://user-images.githubusercontent.com/5672295/53527200-48b73000-3ab4-11e9-9939-15b82149cfe1.png">

### Notes
- I did not remove "Download assets" buttons from the pages where they were previous located—I imagine some folks may have those pages bookmarked or in memory and it seemed unnecessary to remove them.
- I'm using the [jekyll-redirect-from](https://github.com/jekyll/jekyll-redirect-from) Jekyll plugin to redirect renamed / removed pages. It sounds like this should work with Github pages (it's instead of using a `.htaccess`)

## Testing Instructions
* Confirm that  `http://localhost:4000/design/logo.html` -> `http://localhost:4000/design/logo-guidelines.html`
* Confirm that  `http://localhost:4000/identity/open-source.html` -> `http://localhost:4000/identity/products.html`
* Navigate to "Logo downloads" page, and confirm that all logo downloads from "Company", "Fellowships", "Products" and "Community" are located on this page.

Connects #115
